packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2030,6 +2030,7 @@
   "Username_is_already_in_here": "`@%s` is already in here.",
   "Username_is_not_in_this_room": "The user `#%s` is not in this room.",
   "Username_Placeholder": "Please enter usernames...",
+  "Username_already_exist": "Username already exists. Please take some other name.",
```

Comment:
Can we change it to "Username already exists. Please try another username."? I believe that makes more sense.

Reply:
Agreed